exercises/chx_histogram.py

```diff
@@ -0,0 +1,53 @@
+'''
+Author: Jorge Diaz Jr
+Plots histogram for CHX plans + detectors
+'''
+import os
+from skbeam.core.accumulators.histogram import Histogram
+import matplotlib.pyplot as plt
+import pandas as pd
+import numpy as np
+
+plt.ion()
+plt.clf()
+
+def create_dfs(file_path, files):
+    dfs = []
+    for file in files:
+        df = pd.read_csv(file_path +'/'+ file, sep=',')
+        df.index = pd.to_datetime(df.pop('timestamp'))
+        dfs.append(df)
+    return dfs
+
+
+def histogram_plot(dfs):
+    min_size = 0
+    max_size = 1e9 # 10 GB here
+    Nbins = 10000
+    fig, ax = plt.subplots()
+    x_range = np.linspace(0,1e-6, 10000)
+    for df in dfs:
+        h = Histogram((Nbins, min_size, max_size))
+        col_name = df.columns.values[0]
+        col = np.array(df[col_name])
+        h.fill(col)
```

Comment:
can you make a new figure for each?
`fig, ax = plt.subplots()`

Reply:
I did it for both the CHX plans and CHX plans+detectors.